saleor/graphql/order/types.py

```diff
@@ -271,6 +353,24 @@ class OrderLine(CountableDjangoObjectType):
         description="Price of the single item in the order line.",
         required=True,
     )
+    undiscounted_unit_price = graphene.Field(
+        TaxedMoney,
+        description=(
+            "Price of the single item in the order line without applied an order line "
+            "discount."
+        ),
+        required=True,
+    )
+    unit_discount = graphene.Field(
+        Money,
+        description=("The discount applied to the single order line."),
```

Comment:
I think we can drop parentheses:
```suggestion
        description="The discount applied to the single order line.",
```